msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/GuestAccountAcquireTokenNetworkTests.java

```diff
@@ -60,11 +60,11 @@ public void testGetTokenSilentlyForEachGuestTenantSuccess() {
         performSilentAcquireTokenCall(getAccount(), authorityPrefix + labGuest.getHomeTenantId());
 
         // get a token silently for each guest tenant
```

Comment:
Should we make the comment more clear here  'for each tenant the user is a guest'.